internal/storage/debug_bucket.go

```diff
@@ -126,6 +126,12 @@ func (b *debugBucket) Name() string {
 }
 
 func (b *debugBucket) BucketType() gcs.BucketType {
+	id, desc, start := b.startRequest("GetStorageLayout(%q)", b.Name())
+	var err error
+	// In case getStorageLayout call returns error , the bucket type is set to unknown and
```

Comment:
nit:
```
var err error

id, desc, start := b.startRequest("GetStorageLayout(%q)", b.Name())
defer b.finishRequest(id, desc, start, &err)

return b.wrapped.BucketType()
```

Also add comment that bucketType internally calls GetStorageLayout from control client to avoild confusion why we kept `GetStorageLayout` instead of BucketType in log.

Reply:
Done.